pkg/phlare/phlare.go

```diff
@@ -726,19 +727,35 @@ func (f *Phlare) stopped() {
 			level.Error(f.logger).Log("msg", "error closing tracing", "err", err)
 		}
 	}
+	if err := f.logger.buf.Flush(); err != nil {
+		fmt.Println("error flushing logs:", err)
+	}
 }
 
-func initLogger(logFormat string, logLevel dslog.Level) log.Logger {
-	writer := log.NewSyncWriter(os.Stderr)
```

Comment:
There's no particular reason to explicitly synchronise stdout/err writes: writes less than 4K (pipe buffer size) are already synchronous, and large outputs such as goroutine dump are not directed to the writer.